.prettierrc

```diff
@@ -4,5 +4,17 @@
     "singleQuote": true,
     "trailingComma": "all",
     "bracketSpacing": true,
-    "arrowParens": "avoid"
+    "arrowParens": "avoid",
+    "overrides": [
+        {
+          "files": "**/*.sol",
+          "options": {
+            "printWidth": 80,
```

Comment:
120?

Reply:
do you want it 120? This was the default config on the prettier-solidity website.

Reply:
yeah 120!!! (not what i want but that's what we use haha)

Reply:
sg will change 👍